packages/ses/src/compartment-evaluate.js

```diff
@@ -30,8 +29,11 @@ export const provideCompartmentEvaluator = (compartmentFields, options) => {
     let { globalTransforms } = compartmentFields;
     const { globalObject, globalLexicals } = compartmentFields;
 
-    let localObject = globalLexicals;
-    if (__moduleShimLexicals__ !== undefined) {
+    let moduleLexicals;
+    if (
+      __moduleShimLexicals__ !== undefined &&
+      __moduleShimLexicals__ !== null
```

Comment:
This is really one of those cases where I prefer to write `__moduleShimLexicals__ != null`